wAbout/TLSUsers.md

```diff
@@ -0,0 +1,60 @@
+---
+title: TLS Changes
+filename: "TLSUsers.md"
+layout: custom
+keywords: []
+headings: ["Overview"]
+links: []
+image_dir: ""
+images: [
+    {file: "", type: "png", site: "Addin", cat: "", sub: "", report: "", ribbon: "", config: ""}
+]
+description: 
+---
+* * *
+
+## Overview
+
+Microsoft [announced](https://learn.microsoft.com/en-us/lifecycle/announcements/tls-support-ending-10-31-2024){:target="_blank"}{:rel="noopener noreferrer"} that all TLS communications must use version 1.2 starting October 31, 2024. After that date, any communication using TLS 1.0 or 1.1 will not work.
+
+This means users on versions of Interject that use TLS 1.1 will need to either:
+
+- [Update Interject](https://portal.gointerject.com/download-interject.html)
+
+- [Update TLS](/wAbout/TLS.html#set-tls-settings-via-idssettings-file) to TLS 1.2.
+
+### Update Interject From Version 2.5+
+
+When updating Interject from a 2.5 version and above, you simply have to click the **Check for Updates** button on the Interject ribbon. Then click **Get Update**:
+
+![](/images/TLSUser/GetUpdate.png)
+<br>
+
+For detailed instructions about updating Interject, see [here](/wAbout/Updating-INTERJECT.html).
+
+
+### Update Interject From Version 2.3.34
+
+If you have Interject version 2.3.34, you will have to uninstall this version first.
+
+1. [Uninstall](/wAbout/Uninstalling.html) Interject
+
+2. [Download](/wPortal/DownloadInterject.html) Interject from the portal site
+
+3. [Install](/wAbout/SharedComputer.html) Interject
+
+### Updating TLS 1.2
```

Comment:
we can rename this to a header "# Manually Overriding TLS Settings"

Reply:
ok, thought I had that changed